src/components/PromptInfoStorybook.jsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+
+export default function PromptInfo({ triggerPrompt, onClick, prompts }) {
```

Comment:
Why do you have this component twice? Ideally you'd want to import your `../data/prompts.json` in the `App.js` and then pass it as props to this component as you do in this component for storybook. That way you have a single point where data is distributed and you don't have to have two almost identical components just for storybook.

Reply:
Pure frustration is the reason for this one's existence. 😆 I thought we should avoid prop drilling at all costs 😅 but knowing it's fine makes dealing with Storybook a lot easier! 😁 Thank you!

Reply:
Prop drilling is totally fine and should be used where needed. Think about it this way: often you start with data from a `json` file because you don't have a backend yet or something like that. At some point you might want to add the data yourself and build a `Form` component. Now if you haven't done props drilling, you might have to rebuild all your components because you get the data from the `Form` via the `App` component. Does that make sense?